### DISCUSSION TITLE
PnP Provision Template Site title missing??? *HELP*

### DISCUSSION THREAD

**Discussion by cstaylor80** [pnp/powershell#2121] [General]
Hi I'm going out of brain at the moment. I have installed the latest version from pnp to powershell which i wanted to clone a team's site (group and non group enabled) which works fine. I then go to apply this template to another site with all the settings and while it does do it.. the destination site title in the header is missing?????. I have tried everything and exhausted google and have come up with nothing to make this work. The title is there in the site information however its NOT displayed on the home page of the site? why?

As mentioned I have uninstalled via PowerShell PnP and installed the current release and still the same problem?. is there something I am missing here? 

any help would be great as I need to find a way to display the current site name which has disappeared. 

![Screenshot 2022-07-09 102416](https://user-images.githubusercontent.com/24868053/178087119-83a46c44-123f-4f4c-95ff-341991b558cc.png)

**Comment by ZARSYNTEX** [comment]
Same problem here, PS Version 7.2.4.  and 1.11.0     PnP.PowerShell

**Comment by Nathaire** [comment]
Same problem here, PS Version 7.2.6 and 1.11.0 PnP.PowerShell
Tested same template with PnP.PowerShell 1.10.0 -> no issue

**Comment by LadyGwen** [comment]
I had the same problem. After checking the web properties I realized that the web property HideTitleInHeader was set to true (wasn't the case in my origin template site). So after setting it to false it was displayed correctly. Is there a way to export this setting into the PnP template?

**Reply by Ddv0623** [reply]
you can try this one <pnp:Header Layout="Compact" MenuStyle="Cascading" ShowSiteTitle="true" ShowSiteNavigation="true" />

**Comment by BaronSparky** [comment]
Noticed the same with Communication sites using 1.12.0.  I typically just do a "Set-PnPWebHeader -HideTitleInHeader:$false" after the template has been applied, in the script.
